docs/tutorial/get-started.md

```diff
@@ -0,0 +1,243 @@
+a "consumer", in this case is just a `Pod`.
+
+```
+$ kubectl get persistentvolumeclaim
+NAME          STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS             AGE
+data-volume   Pending                                      piraeus-three-replicas   14s
+```
+
+For our consumer, we will create a Deployment for a simple web server, serving files from our volume.
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: web-server
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: web-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: web-server
+    spec:
+      containers:
+        - name: web-server
+          image: nginx
+          volumeMounts:
+            - mountPath: /usr/share/nginx/html
+              name: data
+      volumes:
+        - name: data
+          persistentVolumeClaim:
+            claimName: data-volume
+EOF
+```
+
+After a short wait, the Pod is `Running`, and our `PersistentVolumeClaim` is now `Bound`:
+
+```
+$ kubectl wait pod --for=condition=Ready -l app.kubernetes.io/name=web-server
+pod/web-server-84867b5449-hgdzx condition met
+$ kubectl get persistentvolumeclaim
+NAME          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS             AGE
+data-volume   Bound    pvc-9e1149e7-33db-47a7-8fc6-172514422143   976563Ki   RWO            piraeus-three-replicas   1m
+```
+
+Checking the running container, we see that the volume is mounted where we expected it:
+
+```
+$ kubectl exec deploy/web-server -- df -h /usr/share/nginx/html
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/drbd1000   921M   24K  857M   1% /usr/share/nginx/html
+```
+
+Taking a look with the `linstor` client, we can see that the volume data is replicated on three nodes.
+
+```
+$ kubectl -n piraeus-datastore exec deploy/linstor-controller -- linstor resource list-volumes
++-------------------------------------------------------------------------------------------------------------------------------------------+
+| Node           | Resource                                 | StoragePool | VolNr | MinorNr | DeviceName    | Allocated | InUse  |    State |
+|===========================================================================================================================================|
+| n1.example.com | pvc-9e1149e7-33db-47a7-8fc6-172514422143 | pool1       |     0 |    1000 | /dev/drbd1000 | 16.91 MiB | InUse  | UpToDate |
+| n2.example.com | pvc-9e1149e7-33db-47a7-8fc6-172514422143 | pool1       |     0 |    1000 | /dev/drbd1000 |   832 KiB | Unused | UpToDate |
+| n3.example.com | pvc-9e1149e7-33db-47a7-8fc6-172514422143 | pool1       |     0 |    1000 | /dev/drbd1000 |   832 KiB | Unused | UpToDate |
++-------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Should I explain why the allocated size differs between node? I don't think so, but maybe people get worried?

Reply:
I don't think it is necessary.

Reply:
Removed it entirely, as a user might not even have 3 nodes. Safer to assume a single node cluster. I guess we need to show replication in another tutorial